assets/constants.js

```diff
@@ -441,7 +441,11 @@ var constants = {
 
   "WHITELISTED_COLLECTION_IDS_FOR_SAVING_GUEST_PROGRESS": [],
 
-  "FEEDBACK_SUBJECT_MAX_CHAR_LIMIT": 50,
+  "WHITELISTED_EXPLORATION_IDS_FOR_SAVING_PLAYTHROUGHS": [
+    "umPkwp0L1M0-", "MjZzEVOG47_1", "9trAQhj6uUC2", "rfX8jNkPnA-1",
+    "0FBWxCE5egOw", "670bU6d9JGBh", "aHikhPlxYgOH", "-tMgcP1i_4au",
+    "zW39GLG_BdN2", "Xa3B_io-2WI5", "6Q6IyIDkjpYC", "osw1m5Q3jK41"
+  ],
 
-  "USE_NEW_SUGGESTION_FRAMEWORK": false
+  "FEEDBACK_SUBJECT_MAX_CHAR_LIMIT": 50
```

Comment:
Not sure what this means, don't think I changed it in my PR?

Reply:
Possibly some merge issue. I added the constant a while back. Could you please add it back? Thanks!